static/app/components/groupPreviewTooltip/groupPreviewHovercard.tsx

```diff
@@ -0,0 +1,81 @@
+import {ComponentProps, Fragment, ReactChild} from 'react';
+import {css, useTheme} from '@emotion/react';
+import styled from '@emotion/styled';
+
+import {Body, Hovercard} from 'sentry/components/hovercard';
+import useMedia from 'sentry/utils/useMedia';
+
+interface GroupPreviewHovercardProps extends ComponentProps<typeof Hovercard> {
+  children: ReactChild;
+  className?: string;
+  hide?: boolean;
+}
+
+const GroupPreviewHovercard = ({
+  className,
+  children,
+  hide,
+  ...props
+}: GroupPreviewHovercardProps) => {
+  const theme = useTheme();
+
+  // No need to preview on hover for small devices
+  const shouldNotPreview = useMedia(`(max-width: ${theme.breakpoints.large})`);
```

Comment:
dang when did we get this

Reply:
I think this should be medium
```suggestion
  const shouldNotPreview = useMedia(`(max-width: ${theme.breakpoints.medium})`);
```

Reply:
The cutoff was large before, but I knocked it down to medium because I agree, I think that's better